tests/scripts/all.sh

```diff
@@ -222,9 +222,9 @@ cleanup()
     fi
 
     command make clean
-    cd crypto
-    command make clean
-    cd ..
+    if [ -n "$HAVE_CRYPTO_SUBMODULE" ]; then
+        command make -C crypto clean
```

Comment:
This conflicts with https://github.com/ARMmbed/mbedtls/pull/2438 which has similar fixes. Also, I fail to see the connection with the main purpose of this PR (address Pylint issues). Please consider removing this commit from the PR in the interest of having single-purpose PRs.

Reply:
This is the third time I've been annoyed by having to check out the submodule even though it was irrelevant to what I was doing, hence I got rid of the submodule obligation. But ok, I'll remove it from this PR.